example/main.go

```diff
@@ -2,14 +2,22 @@ package main
 
 import (
 	"fmt"
+	"path/filepath"
 
-	"github.com/getlantern/pacon"
+	"github.com/getlantern/pac"
 )
 
 func main() {
-	pacon.PacOn("localhost:12345/pac")
-	fmt.Println("proxy set, any key to continue...")
+	helperFullPath, _ := filepath.Abs("./pac-cmd")
+	iconFullPath, _ := filepath.Abs("./icon.png")
+	pac.EnsureHelperToolPresent(helperFullPath, "Input your password and save the world!", iconFullPath)
+	err := pac.On("localhost:12345/pac")
+	if err != nil {
+		fmt.Printf("Error set proxy: %s\n", err)
+		return
+	}
+	fmt.Println("proxy set, Enter continue...")
 	var i int
 	fmt.Scanf("%d\n", &i)
-	pacon.PacOff()
+	pac.Off()
```

Comment:
Checking varies among OS and requires UI interaction on Mac. I think it's not worth for a stable library.